trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/map-to-selected-payment-details-filters-view-model.test.ts

```diff
@@ -0,0 +1,142 @@
+import { CURRENCY } from '@ukef/dtfs2-common';
+import { when } from 'jest-when';
+import { mapToSelectedPaymentDetailsFiltersViewModel } from './map-to-selected-payment-details-filters-view-model';
+import { getPaymentDetailsTabHref } from './get-payment-details-tab-href';
+
+jest.mock('./get-payment-details-tab-href');
```

Comment:
Can we call the helper please instead of mocking and hard coding the assertion? i.e

```ts
const expected = getPaymentDetailsTabHref({ ... });

expect(result!.paymentCurrency!.removeHref).toEqual(expected);
```

instead of:

```ts
expect(result!.paymentCurrency!.removeHref).toEqual('/remove-payment-currency-filter');
```

Reply:
Has this been agreed as the pattern going forwards? I think this makes it harder to see the error going forwards if this test were ever to fail because the test output will display a long string that was generated by a function not in this file?

Reply:
Ok will change.